cmd/cli/from_2.3_to_2.4.go

```diff
@@ -14,7 +14,7 @@ func updateFrom23(logger moira.Logger, dataBase moira.Database) error {
 	return nil
 }
 
-func downgradeTo23(logger moira.Logger, dataBase moira.Database) error {
+func downgradeTo23() error {
```

Comment:
Может оставить хотя бы в виде сигнатуры, чтобы у всех методов переезда туда-сюда был одинаковый интерфейс?

```suggestion
func downgradeTo23(moira.Logger, moira.Database) error {
```

Reply:
В идеале ещё коммент о том, почему эта функция пустая

Reply:
`В идеале ещё коммент о том, почему эта функция пустая`

какой?)

Reply:
Я не знаю, так бы сразу написал :sweat_smile: 

Это какие-то технологии предков, 6 лет назад написан код